torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13576,7 +13576,7 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
                DecorateInfo(unittest.skip("Skipped!"), 'TestFakeTensor', 'test_fake_crossref_backward_no_amp'),
            )),
     OpInfo('sparse.mm',
-           dtypes=floating_types_and(torch.bfloat16),
+           dtypes=floating_types_and(torch.bfloat16, torch.float16),
```

Comment:
This is more of a general question, but why do we only run sparse_mm tests on CPU, don't we also support sparse.mm on CUDA to some extend too now? Should we remove the `onlyCPU` decorators and args to OpInfo?